domains/sajidhussain.json

```
@@ -0,0 +1,10 @@
+{
+    "repo": "https://github.com/Sajid-tech/sajid-tech.github.io",
+    "owner": {
+        "username": "Sajid-tech",
+        "email": "s.khan9430319425@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://sajid-tech.github.io"
```

`CNAME` should not contain protocol
```suggestion
        "CNAME": "sajid-tech.github.io"
```

---

okay updated thej json